docs/docs/sourcing-from-etsy.md

```diff
@@ -0,0 +1,116 @@
+
+```js:title=gatsby-config.js
+plugins: [
+  {
+    resolve: "gatsby-source-etsy",
+    options: {
+      // highlight-start
+      apiKey: "your api key here",
+      shopId: "your shop id or shop name here",
+      // highlight-end
+      language: "en", // optional
+    },
+  },
+],
+```
+
+### Testing your queries
+
+Once you have everything set up, run `gatsby develop` to start your site locally. At that point, you should be able to query for data related to your featured listings in [GraphiQL](/docs/running-queries-with-graphiql/). As an example, the following query gets the total number of featured listings in the shop as well as the price, title, and description of each item.
+
+```graphql
+allFeaturedEtsyListing {
+  totalCount
+    nodes {
+      price
+      title
+      description
+    }
+  }
+```
+
+Make sure you have _featured_ items! To edit your store, go to the Shop Manager and select your Etsy shop under "Sales Channels". You should see an option to edit or enable featured items.
+
+## Displaying Etsy listings
+
+You can pull your Etsy listing data into a page or component by composing a GraphQL query and specifying the data you would like to display. For example, consider the following query, which gets the title, price, URL, and an image of each featured listing.
+
+```graphql
+allFeaturedEtsyListing {
+  nodes {
+    id
+    title
+    price
+    url
+    description
+    childEtsyListingImage {
+      childFile {
+        childImageSharp {
+          fixed(width: 400, height: 400) {
+            ...GatsbyImageSharpFixed
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+Using this query on a page or in a component will allow you to display your listings. The following example loops through the featured items and displays them within a component.
+
+```jsx
+{
+  data.allFeaturedEtsyListing.nodes.map(item => (
+    <section key={item.id}>
+      <a href={item.url}>
+        <h2>{item.title}</h2>
+      </a>
+      <Img fixed={item.childEtsyListingImage.childFile.childImageSharp.fixed} />
+      <p>${item.price}</p>
+      <p>{item.description}</p>
+    </section>
+  ))
+}
+```
+
+Note that, in this example, there is no alt text passed to the `Img` component because this image can be considered presentational. Etsy doesn't provide space for you specify [text alternatives](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#text-alternatives) for your images. Instead, when you list an item on Etsy, alt text is generated for each image you associate with that item from its title. Including the title or description as alt text here would only repeat the exact same information for anyone using a screen reader.
```

Comment:
```suggestion
Note that, in this example, there is no alt text passed to the `Img` component because this image can be considered presentational. All images appear in association with their title and description. Therefore, including the same information as alt text would be repetitive for anyone using a screen reader.
```
I had a slight rewording for this and then I wondered whether it makes more sense to ignore why Etsy doesn't do this and only mention the example above? So that's the recommendation I made, but open to alternatives.